app/views/users/new.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Login Form </h1>
\ No newline at end of file
+<!-- <form action="/users" method="post" accept-charset="UTF-8">-->
+<!--  Username: <input type="text" name="username"><br><br>-->
+<!--  Email: <input type="email" name="email"><br><br>-->
+<!--  Password: <input type="password" name="password"><br>-->
+<!--  <input type="submit" value="Submit">-->
+<!--  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">-->
+<!--</form> -->
+
+<!-- <%= form_tag('/users') do %>-->
+<!-- Username: <div><%= text_field_tag 'username' %></div>-->
+<!-- Email: <div><%= email_field_tag 'email' %></div>-->
+<!-- Password: <div><%= password_field_tag 'password' %></div>-->
+<!-- Submit: <div><%= submit_tag 'submit' %></div>-->
+<!-- <% end %>-->
+
+    <%= form_for :@user do |f| %>
```

Comment:
Because of `:@user`, creating user is not possible.
Change `:@user` to **`@user`**